3party/pugixml/src/utils.hpp

```diff
@@ -0,0 +1,26 @@
+#include "pugixml.hpp"
+
+#include <sstream>
+#include <string>
+#include <utility>
+
+namespace pugi
+{
+template <typename... Args>
+inline std::string XMLToString(pugi::xml_node const & n, Args &&... args)
```

Comment:
No need to mark template as inline. Generally speaking, it's not a good idea to keep patches to pugi here. Could you please move this code? For example, to coding.